compose/ui/ui-uikit/src/nativeInterop/cinterop/utils.def

```diff
@@ -1,3 +1,5 @@
 linkerOpts = -framework UIKit
 package = androidx.compose.ui.uikit.utils
-language = Objective-C
\ No newline at end of file
+language = Objective-C
+compilerOpts = -D_Float16=short
+headerFilter = CMP*
```

Comment:
For now it is by design to include all headers from CMP library. No every header by Obj-C standards should have CMP prefix. This line should be removed.

Reply:
The reason `headerFilter` is needed is that otherwise cinterop leaks platform API from the new headers in the sysroot (see https://youtrack.jetbrains.com/issue/KT-71694). 

Unfortunately, even with the `headerFilter` some platform API leaks, although without producing any side effects in the commonizer (e.g. https://youtrack.jetbrains.com/issue/KT-71828). I don't know why, but I will investigate and talk to K/N team this week.

Reply:
I see. I will try to put headers into a separate directory after the merge.